### PR TITLE
eos: Add new states to eos_vlans

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

##### SUMMARY
Add new states gathered , rendered and parsed to eos_vlans.

related to #23

### CODE REVIEW

File: plugins/modules/eos_vlans.py [hunk 1]
```diff
@@ -59,6 +60,15 @@
         - ID of the VLAN. Range 1-4094
         type: int
         required: true
+      running_config:
+        description:
+        - This option is used only with state I(parsed).
+        - The value of this option should be the output received from the NX-OS device by executing
```
Comment (NilashishC) [inline plugins/modules/eos_vlans.py:66]:
```suggestion
        - The value of this option should be the output received from the EOS device by executing
```

**Comment by GomathiselviS** [comment]
recheck

File: tests/integration/targets/eos_vlans/tasks/main.yaml [hunk 2]
```diff
@@ -1,17 +1,7 @@
...
-
-- name: run test cases (connection=ansible.netcommon.network_cli)
-  include: '{{ test_case_to_run }} ansible_connection=ansible.netcommon.network_cli'
-  with_items: '{{ test_items }}'
-  loop_control:
-    loop_var: test_case_to_run
-  tags: connection_network_cli
+- include: cli.yaml
+  tags:
+    - cli
```
Comment (pabelanger) [inline tests/integration/targets/eos_vlans/tasks/main.yaml:4]: network_cli

File: tests/integration/targets/eos_vlans/tasks/main.yaml [hunk 3]
```diff
@@ -1,17 +1,7 @@
...
+- include: eapi.yaml
+  tags:
+    - eapi
```
Comment (pabelanger) [inline tests/integration/targets/eos_vlans/tasks/main.yaml:7]: httpapi

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Pull request merged**